baseband/vlbi_base/payload.py

```diff
@@ -37,12 +37,14 @@ class VLBIPayloadBase(object):
     """
     # Possible fixed payload size.
     _size = None
-    # Default type for encoded data
+    # Default type for encoded data.
     _dtype_word = np.dtype('<u4')
     """Default for words: 32-bit unsigned integers, with lsb first."""
     # To be defined by subclasses.
     _encoders = {}
     _decoders = {}
+    # Placeholder for sample shape named tuple.
+    _sample_shape_cls = tuple
```

Comment:
Looking at this again, I realize `tuple` is not really a good analogy to a namedtuple instance, since one cannot do `tuple(*a)` (which also leads to the try/except in `fromdata`). I think we should write instead:
```
# outside of class
def _sample_shape_tuple(*sample_shape):
    return sample_shape
```

_sample_shape_cls = _sample_shape_tuple
```
(the construct of `*sample_shape` is guaranteed to be turned into a tuple already). Alternatively, you could directly define the function here.

Reply:
Maybe even easier would be to do
```
_sample_shape_cls = None
```

and then below
```
if self._sample_shape_cls is not None:
    sample_shape = self._sample_shape_cls(*sample_shape)
```

Reply:
Rename as `_sample_shape_maker`?